gammapy/maps/hpxnd.py

```diff
@@ -648,3 +649,28 @@ def get_angle(x, t):
         ax.coords.grid(color='w', linestyle=':', linewidth=0.5)
 
         return fig, ax, p
+
+    def make_region_mask(self, region, inside=True):
+        """Create a mask of a given region
+
+        TODO: implement list of region for each axis
+
+        Parameters
+        ----------
+        region :  `~regions.SkyRegion` object
+            A region on the sky could be defined in pixel or sky coordinates.
+            Note that `~regions.PixelRegion` does not work for healpix maps.
```

Comment:
As mentioned by @cdeil we cannot really support HPX at the moment

Reply:
OK will remove it then.